nectar/src/network.rs

```diff
@@ -90,19 +96,6 @@ impl Nectar {
     pub fn peer_id(&self) -> PeerId {
         PeerId::from(self.identity.public())
     }
-
-    fn poll<BIE>(
-        &mut self,
-        _cx: &mut Context<'_>,
-        _params: &mut impl PollParameters,
-    ) -> Poll<NetworkBehaviourAction<BIE, BehaviourOutEvent>> {
-        if let Some(event) = self.events.pop_front() {
-            return Poll::Ready(NetworkBehaviourAction::GenerateEvent(event));
-        }
-
-        // We trust libp2p to poll us.
-        Poll::Pending
-    }
 }
 
 impl libp2p::swarm::NetworkBehaviourEventProcess<orderbook::BehaviourOutEvent> for Nectar {
```

Comment:
These can go away now :)

Reply:
+ the `events` field as well but I couldn't comment on it :D

Reply:
Ok but how can I be sure it still works afterwards as the network is not tested? I guess your e2e tests will tell me.

Reply:
I just want to be clear that I blindly following your suggestion here :)

Reply:
> Ok but how can I be sure it still works afterwards as the network is not tested? I guess your e2e tests will tell me.

The libp2p project contains a test for this behaviour :)